pftools/python/parflow/tools/io.py

```diff
@@ -26,11 +26,11 @@
     from numba import jit
 except ImportError:
     # Some systems may not have numba capabilities
-    def jit(self, function, *args, **kwargs):
+    def jit():
```

Comment:
Maybe to be on the safe side `def jit(*args, **kwargs):`

Reply:
Sure, I could make that change. Should I do another release? 1.3.2 contains the version in this PR: `def jit():`

Reply:
No I think you are good for now. It is just needed if ever other use of the @jit() decorator is happening. That way we would not care if args or kwargs are passed. So far, the code is not passing anything, but things may change down the road.